jpa/hibernate5-legacy/src/main/java/org/jboss/as/jpa/hibernate5/infinispan/DeprecatedInfinispanRegionFactory.java

```diff
@@ -43,7 +45,7 @@
     @Override
     public void start(SessionFactoryOptions settings, Properties properties) throws CacheException {
         Logger.LOGGER.deprecatedRegionFactory(this.getClass().getName(), this.getClass().getSuperclass().getSuperclass().getName(), SHARED, this.shared);
-        properties.setProperty(SHARED, this.shared);
+        properties.put(SHARED, this.shared);
```

Comment:
What's the purpose of this change? These should do the same thing.

Reply:
Just part of code cleanup.

Reply:
I'm curious on how this is a cleanup though. It's using `put` on a `java.util.Properties` type. Using `setProperty()` seems more appropriate to me.

Reply:
Sorry, I was referring to a different change that was already merged.  For this change, we can revert it, the equivalent class for ORM 5.3 integration is now passed a Map, so this code is kind of line of line identical with that but doesn't need to be.   I'll push this change in a few minutes after testing it.